todo_list/todo.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="todo.css" />
+    <title>Todo_list</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Alkalami&family=Gaegu&family=Jua&family=Nanum+Pen+Script&family=Stylish&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/06176f9978.js" crossorigin="anonymous"></script>
+</head>
+<body>
+<!-- 헤더 -->
+    <header>
+        <span class="header_title">To-do List</span> <br>
+        <span class="sub_title">Plan your day. It's gonna be a great day</span>
+    </header>
+
+<!-- 네비게이션 바 -->
+    <nav>
+        <button id="today">오늘만 보기</button>
+        <button id="tomorrow">내일만 보기</button>
+        <button id="togather">함께 보기</button>
+    </nav>
```

Comment:
요기 `togather`로 오타가 났네요~
세 개의 버튼의 디자인이 중복되기 때문에 중복사용이 불가능한 id 보다는 class 속성을 주는 게 나을 것 같아요.
together 버튼의 디자인만 바꾸고 싶다면 [:last-child()](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child)와 같은 선택자를 이용할 수 있어요.
```suggestion
    <nav>
        <button class="nav__button" type="button">오늘만 보기</button>
        <button class="nav__button" type="button">내일만 보기</button>
        <button class="nav__button" type="button">함께 보기</button>
    </nav>
```
```css
  nav > nav__button:last-child(){
   // 함께 보기 버튼만 갖고 있는 디자인 적용
 }
```
이런 식으로 변경하면 좋을 것 같아요.

그리고 button 태그에는 type을 지정해주어야 해요. type을 정해주지 않으면 디폴트가 submit이라 오작동할 수도 있어요.